utility/session.go

```diff
@@ -0,0 +1,138 @@
+package utility
+
+import (
+	"encoding/binary"
+	"encoding/hex"
+	"github.com/pokt-network/pocket/shared/crypto"
+	"github.com/pokt-network/pocket/shared/modules"
+	"github.com/pokt-network/pocket/utility/types"
+)
+
+type Session interface {
+	NewSession(sessionHeight int64, blockHash string, geoZone GeoZone, relayChain RelayChain, application modules.Actor) (Session, types.Error)
```

Comment:
Each session is `mono-chain and mono-geo`

This is fundamental as it would create even more possible combinations of sessions and increase computational complexity during block production and servicing

Reply:
Can you add this to the Q&A?